src/controllers/update/overseas.entity.update.details.controller.ts

```diff
@@ -52,10 +53,16 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     logger.debugRequest(req, `POST OVERSEAS_ENTITY_UPDATE_DETAILS`);
```

Comment:
We've been using logger.debugRequest(req, `${req.method} ${req.route.path}`);

Reply:
Are we going to retrospectively change this everywhere? Barry and I left it as there are some other update controllers using the "old" approach that probably won't be touched in the near future. I've updated it now but perhaps a ticket to update the rest is needed.